managed/Makefile

```diff
@@ -11,7 +11,8 @@ help:                           ## Display this help message.
 
 # `cut` is used to remove first `v` from `git describe` output
 PMM_RELEASE_PATH ?= ../bin
-PMM_RELEASE_VERSION ?= $(shell git describe --always --dirty | cut -b2-)
+#PMM_RELEASE_VERSION ?= $(shell git describe --always --dirty | cut -b2-)
+PMM_RELEASE_VERSION ?= 2.33.0
```

Comment:
Included in the commit by mistake?

Reply:
Yes and no. It's crucial for me at the moment and I'll remove it once I'll be ready for the review =) Thanks for pointing out!